sky/tools/gn

```diff
@@ -126,13 +120,10 @@ def parse_args(args):
   args = args[1:]
   parser = argparse.ArgumentParser(description='A script run` gn gen`.')
 
-  parser.add_argument('--debug', default=True, action='store_true')
-  parser.add_argument('--release', default=False, dest='debug', action='store_false')
+  parser.add_argument('--debug', default=False, action='store_true')
+
+  parser.add_argument('--runtime-mode', type=str, choices=['develop', 'profile', 'release'], default='develop')
```

Comment:
These names don't quite line up with these names: https://github.com/flutter/flutter/wiki/Flutter's-modes

We should make them consistent, either by changing the wiki or changing the names here.  @Hixie